jib-build-plan/scripts/prepare_release.sh

```diff
@@ -39,8 +39,8 @@ if [[ $(git status -uno --porcelain) ]]; then
   Die 'There are uncommitted changes.'
 fi
 
-# Runs integration tests.
-./gradlew :jib-build-plan:integrationTest --info --stacktrace
+# Runs checks and integration tests.
+./gradlew :jib-build-plan:check :jib-build-plan:integrationTest --info --stacktrace
```

Comment:
Quick question: `kokoro/continuous.sh` runs `build` and `integrationTest`. What's the difference between `build` and `check`?

Reply:
yeah I should've mentioned, `build`  already runs `check` (but not `integrationTest`, we always have had integration test be explicit)

Reply:
`check` runs all tests, checkstyle and format things